maps/away/away_site/crystal_planet_outpost/crystal_planet_outpost_.dm

```diff
@@ -9,7 +9,7 @@
 	suffixes = list("crystal_planet_outpost.dmm")
 
 	sectors = list(ALL_POSSIBLE_SECTORS)
-	sectors_blacklist = list(SECTOR_BURZSIA, SECTOR_HANEUNIM, SECTOR_SRANDMARR, SECTOR_TAU_CETI) //it's a whole planet, shouldn't have it in predefined sectors
+	sectors_blacklist = list(SECTOR_BURZSIA, SECTOR_HANEUNIM, SECTOR_SRANDMARR, SECTOR_TAU_CETI, SECTOR_UUEOAESA) //it's a whole planet, shouldn't have it in predefined sectors
```

Comment:
at this point there should be a a list of sectors named like `ALL_SPECIFIC_SECTORS` or something, documented to mean `/// all non-generic, named and specific sectors, where generic planets or the like should not spawn`

instead blacklisting specific sites every time one is added